machineEnvironment.sh

```diff
@@ -45,13 +44,15 @@ if [ "`hostname | grep daint`" != "" ] ; then
     . /opt/modules/default/init/bash
     . /etc/bash.bashrc.local
     export host="daint"
-    scheduler="slurm"
     queue="normal"
     nthreads=8
     mpilaunch="srun"
     installdir=/project/d107/install/
     container_engine="sarus"
     export CUDA_ARCH=sm_60
+    if [ -z "${scheduler}" ] ; then
```

Comment:
I agree with Rhea's comment, that this is not super nice. Basically, the intent of this variable is to say what different machines have. And daint has a scheduler and it's SLURM. So conditionally setting this a) changes behavior as compared to other values set here and b) introduces the strange behavior where the scheduler that daint has is only SLURM in case the user does not think differently.
I actually think what we want to have is an option to the `run_command` to override the default scheduler being used.